examples/displaywindow.html

```diff
@@ -36,5 +46,45 @@
       title="'Window 2 - A powerful window'">
     </ngeo-displaywindow>
 
+    <ngeo-displaywindow
+            class="window3"
+            clear-on-close="::false"
+            content-template="::ctrl.window3Template"
+            content-scope="::ctrl.windowScope"
+            draggable="::true"
+            resize="::true"
```

Comment:
If draggable AND resize are `true`, that's the same as `desktop=true`.
And by default, desktop is `true`.
So you can remove both binding, that will be the same.
(read the doc :-p )

I think you can also remove height and width, that's not what we want to show (=noise).

Reply:
Window2 has both as well... But I agree on leaving out stuff, we don't want to show. I will keep the width/height on the last one though, otherwise you need to resize the window first to see what I want to show.

Reply:
Yes, but in windows 2 I want to demonstrate it.
But ok, you can let it.